src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FtpaRespondentPreparerTest.java

```diff
@@ -66,8 +59,8 @@ void should_set_out_of_date_submission_state_no() {
         when(callback.getEvent()).thenReturn(Event.APPLY_FOR_FTPA_RESPONDENT);
         when(caseDetails.getCaseData()).thenReturn(asylumCase);
         when(dateProvider.now()).thenReturn(LocalDate.now());
-        final String appealDate = dateProvider.now().minusDays(1).toString();
-        when(asylumCase.read(APPEAL_DATE)).thenReturn(Optional.of(appealDate));
+        final String ftpaApplicationDeadline = dateProvider.now().plusDays(5).toString();
```

Comment:
preferably use static final fields instead of numbers. Like IN_TIME_VALID_DAYS_UK or something similar.